### PR TITLE
Updated changelog format

### PR DESCRIPTION
**Pull request by owais**
Update changelog format to comply with GDI spec

### CODE REVIEW

**Review by seemk — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,83 +1,73 @@
 # Changelog
 
-## [Unreleased]
+## Unreleased
 
-## [0.17.0] - 2021-08-29
-### Changed
+## 0.17.0 - 2021-08-29
+### General
```
Comment (pellared) [inline CHANGELOG.md:6]: there is a missing blank line in betwen

File: CHANGELOG.md [hunk 2]
```diff
@@ -87,71 +77,71 @@
...
 - Allow all compatible versions for stable packages (API, SDK, exporters, propagators)
   and lock to exact version for unstable ones (instrumentations).
   ([#35](https://github.com/signalfx/splunk-otel-python/pull/35))
 
-### Removed
-
-- Removed support for `--exporters` CLI flag from `splunk-py-trace-bootstrap` command.
-  [#34](https://github.com/signalfx/splunk-otel-python/pull/34)
+## 0.10.0 - 03-27-2021
+### General
```
Comment (pellared) [inline CHANGELOG.md:123]: there is a missing blank line in betwen

File: CHANGELOG.md [hunk 3]
```diff
@@ -87,71 +77,71 @@
...
 
-### Added
+### Breaking Changes
 
-- Added support for `--access-token` and `--service-name` to `splunk-py-trace` command.
+- `start_tracing()` was moved from `splunk_otel.tracing` to `splunk_otel`.
   [#34](https://github.com/signalfx/splunk-otel-python/pull/34)
+- Removed support for `--exporters` CLI flag from `splunk-py-trace-bootstrap` command.
+  [#34](https://github.com/signalfx/splunk-otel-python/pull/34)
+### Enchancements
```
Comment (pellared) [inline CHANGELOG.md:111]: there is a missing blank line in betwen

File: CHANGELOG.md [hunk 4]
```diff
@@ -87,71 +77,71 @@
...
   [#58](https://github.com/signalfx/splunk-otel-python/pull/58)
 - `telemetry.auto.version` will now correctly refer to `opentelemetry-instrumentation` version being used.
   [#67](https://github.com/signalfx/splunk-otel-python/pull/67)
-
-### Deprecated 
-
 - Deprecated support for `SPLUNK_SERVICE_NAME` environment variable.
   Please use `OTEL_SERVICE_NAME=<my-service-name>` instead.
   [#57](https://github.com/signalfx/splunk-otel-python/pull/57)
+### Enhancements
```
Comment (pellared) [inline CHANGELOG.md:87]: there is a missing blank line in betwen

**Review by pellared — COMMENTED** [review]
some "linting" that `markdownlint` would recommend

**Pull request merged**